Source/CarthageKit/Git.swift

```diff
@@ -57,6 +57,30 @@ public struct GitURL: Equatable {
 		}
 	}
 
+	/// Amount of time before a git repository is fetched again. Defaults to 1 minute
+	public static var fetchCacheInterval: NSTimeInterval = 60.0
```

Comment:
So as discussed above due to the nature of how we fetch on demand there's many different paths into these fetch calls. So is making it a parameter to the funcs of the GitURL, but still a static var accessed in the two methods below (clone/fetch) ok, or should it get pulled out higher. At what point should it switch from a parameter to the global being passed in